vendor/engines/split_accounts/app/services/split_accounts/attribute_splitter.rb

```diff
@@ -14,6 +14,10 @@ def split(original_object, split_object, split)
       end
     end
 
+    def to_ary
```

Comment:
Fair question. `to_ary` is actually a standard ruby method. `Array(xxx)` will call `to_ary` under the hood if it supports it. `nil` implements it as `[]` and `Array` implements it as `self`.